File: ejercicios/descuento.py
if __name__ == '__main__':
	print("¿Cuál es tu nombre?")
	nombre = input()
	print("¿Cuántos escritorios deseas comprar?")
	cantidad = float(input())
	cantidad1 = cantidad*800000
	if cantidad<=5:
		descuento = cantidad1*0.1
		print(nombre," ,tu pago con descuento será de: ",descuento)
	else:
		if cantidad>=5 and cantidad<10:
			descuento = cantidad1*0.2
			print(nombre," ,tu pago con descuento será de: ",descuento)
		else:
			if cantidad>=10:
				descuento = cantidad1*0.4
				print(nombre," ,tu pago con descuento será de: ",descuento)

File: ejercicios/descuento.cpp
#include <iostream>

int main() {
	float cantidad;
	float cantidad1;
	float descuento;
	float nombre;
	cout << "¿Cuál es tu nombre?" << endl;
	cin >> nombre;
	cout << "¿Cuántos escritorios deseas comprar?" << endl;
	cin >> cantidad;
	cantidad1 = cantidad*800000;
	if (cantidad<=5) {
		descuento = cantidad1*0.1;
		cout << nombre << " ,tu pago con descuento será de: " << descuento << endl;
	} else {
		if (cantidad>=5 && cantidad<10) {
			descuento = cantidad1*0.2;
			cout << nombre << " ,tu pago con descuento será de: " << descuento << endl;
		} else {
			if (cantidad>=10) {
				descuento = cantidad1*0.4;
				cout << nombre << " ,tu pago con descuento será de: " << descuento << endl;
			}
		}
	}
	return 0;
}
